flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/sink/TestAvroGenericRecordToRowDataMapper.java

```diff
@@ -18,10 +18,11 @@
  */
 package org.apache.iceberg.flink.sink;
```

Comment:
I would have expected to include other test files from the `sink` package here is well, but just picking 2 files seems quite random to me. I'd suggest to focus on one or more packages per PR (depending on how big the changes are) instead of just randomly picking files across the codebase